src/main/java/io/wispforest/owo/ui/component/EntityComponent.java

```diff
@@ -72,7 +73,7 @@ protected EntityComponent(Sizing sizing, EntityType<E> type, @Nullable NbtCompou
         this.dispatcher = client.getEntityRenderDispatcher();
         this.entityBuffers = client.getBufferBuilders().getEntityVertexConsumers();
 
-        this.entity = type.create(client.world);
+        this.entity = type.create(client.world, SpawnReason.BREEDING);
```

Comment:
any reason for this particular spawn reason, or is it just arbitrary?
if there is a reason why it has to be this specifically it would be nice to have it written as a comment

Reply:
Because it was funny

Reply:
Understandable